httpbakery/form/form.go

```diff
@@ -49,11 +50,17 @@ client's responsibility to interpret the schema and present it to the
 user.
 */
 
-// Filler represents an object that can fill out a form. The given schema
-// represents the form template. The returned value should be compatible
-// with this.
+// Form describes a form that needs to be completed to perform the login.
+type Form struct {
+	Title  string
+	Fields environschema.Fields
+}
+
+// Filler represents an object that can fill out a form. The the form is
+// described in f. The returned value should be compatible with the
+// schema defined in f.Fields.
 type Filler interface {
```

Comment:
It crossed my mind that we should be using environschema/form.Filler before, but now I'm more sure of that.